packages/esm-appointments-app/translations/en.json

```diff
@@ -117,6 +117,7 @@
   "selectVisitType": "Please select a Visit Type",
   "service": "Service",
   "serviceType": "Service Type",
+  "serviceWorkloadTitle": "",
```

Comment:
An empty translation 🤔

Reply:
It came out after extracting translations.